duckduckgo/Android
# Allow target _blank links to open in a new tab

Task/Issue URL: https://github.com/duckduckgo/Android/issues/502
Tech Design URL: N/A
CC: 

**Description**:
The default behavior is to open a new tab after click on `target="_blank"` links. multiple window support has been enabled in the webview settings

**Steps to test this PR**:
1. Open url https://www.w3schools.com/tags/att_a_target.asp and click link "try it yourself" with target _blank attribute
1. See new tab added in ddg browser 

![ddg-issue502](https://user-images.githubusercontent.com/8261416/59556190-58e91180-8fb6-11e9-84b9-3e54629fb27f.gif)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
